src/plugins/notebook/components/Notebook.vue

```diff
@@ -728,6 +734,39 @@ export default {
             this.selectPage(pageId);
 
             this.syncUrlWithPageAndSection();
+        },
+        activeTransaction() {
+            return this.openmct.objects.getActiveTransaction();
+        },
+        startTransaction() {
+            this.openmct.objects.startTransaction();
+        },
+        saveTransaction() {
+            const transaction = this.activeTransaction();
+
+            if (!transaction) {
+                return;
+            }
+
+            return transaction.commit()
```

Comment:
missing `;` at end 😄

Reply:
we've got the catch and finally after it, it's at the end of those